deployments/jwebbinar/image/environments/jwebbinar/global_bashrc

```diff
@@ -0,0 +1,6 @@
+export PS1="${debian_chroot:+($debian_chroot)}\u@notebook:\w\$ "
```

Comment:
I was wondering if we should delete the  jwebbinar enviromment's global_bashrc and just keep the one in the image directory...  Or maybe put it in the environments root directory since it's global.

Reply:
Which would you prefer?  Also, I'm curious about your reasoning.